### ISSUE TITLE
Use language if original post when replying to someone

### ISSUE THREAD

**Issue by SteffoSpieler** [mastodon/mastodon-android#851]
When you answer a post, the language gets reset to your default language without any obvious warning about it. So if you answer e.g. on a German Post, the post language suddenly switches to e.g. English while the Person replying still writes in German, thus making translations not available for English People (as it's marked as English).

**Comment by Some1withgithub** [comment]
Good idea. I have the same problem and opened an issue at the ios repo (where it should be according to mastdodons team?).
see: [ [Feature]: convenient post language buttons #1239 ](https://github.com/mastodon/mastodon-ios/issues/1239#issue-2152315826)
I also thought about having an easy one click switch next to the globe that is used to change the language, where i could switch fast between 2 languages, but that would also require one more click than having to only press the "send" arrow button.

I think it would be a great addition, if reply language would automatically default to the language of the message you are replying to, but than there is still the issue for the op to set the language correctly.
So still always have an indication about which language is active(as it is apparently for ios) and make it easier to set the language.
Sadly chatgpt couldn't tell me where in the source code i could find the place to make this myself, so i have to invest more time to get it right, for me at least.
